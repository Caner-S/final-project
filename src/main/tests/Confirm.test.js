import React from 'react';
import renderer from 'react-test-renderer';

import Confirm from "../components/Confirm";

it('renders correctly when there are no items', () => {
    const tree = renderer.create(<Confirm />).toJSON();
    expect(tree).toMatchSnapshot();
});