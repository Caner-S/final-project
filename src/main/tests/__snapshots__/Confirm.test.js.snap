// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    />
  </button>
</div>
`;
